--  Remove license fields from pyproject.toml as they are currently in an invalid format
--  for newer setuptools, which causes the build to fail.

--- pyproject.toml.orig	2026-07-21 12:58:31 UTC
+++ pyproject.toml
@@ -4,8 +4,6 @@ version = "0.61.7"
 [project]
 name = "json_repair"
 version = "0.61.7"
-license = "MIT"
-license-files = ["LICENSE"]
 authors = [
   { name="Stefano Baccianella", email="4247706+mangiucugna@users.noreply.github.com" },
 ]
